src/bp-core/classes/class-bp-email-tokens.php

```diff
@@ -3413,4 +2580,231 @@ public function token__commenter_url( $bp_email, $formatted_tokens, $tokens ) {
 		return $commenter_url;
 	}
 
+	public function get_email_media( $meta_id, $tokens, $type = 'activity' ) {
```

Comment:
```suggestion
	public function get_email_media( $object_id, $tokens, $type = 'activity' ) {
```

Also please add docblock.
Added pre-check for the object_id and type empty then return.